packages/csv-parser-order/src/parsers/add-return-info.js

```diff
@@ -0,0 +1,104 @@
+import JSONStream from 'JSONStream'
+import { filter, find } from 'lodash'
+import AbstractParser from './abstract-parser'
+
+export default class AddReturnInfoParser extends AbstractParser {
+  constructor (config) {
+    super(config, 'returnInfo')
+  }
+
+  parse (input, output) {
+    this.logger.info('Starting Return Info CSV conversion')
+    return new Promise((resolve, reject) => {
+      const stream = this._streamInput(input)
+        .reduce([], AddReturnInfoParser._reduceOrders)
+        .stopOnError(reject)
+        .pipe(JSONStream.stringify(false))
+        .pipe(output)
+
+      stream.on('finish', resolve)
+      stream.on('error', reject)
+
+      // process.stdout does not emit finish stream
+      if (output._type === 'tty')
```

Comment:
Can you point a reference to where `_type` is documented? I checked nodejs official docs but could not see anything...